security-analysis/security-analysis-default/src/main/java/com/powsybl/security/impl/DefaultSecurityAnalysis.java

```diff
@@ -4,7 +4,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-package com.powsybl.security;
+package com.powsybl.security.impl;
```

Comment:
Should we rename also the package to "default" ?

Reply:
"default" is a keyword, we cannot use it as package name

Reply:
Ah ... that's an issue indeed ^^ go for impl then (workarounds like `default_` or `defaultimpl` will not be prettier)